apps/www/components/Auth/CodeAuthorization.js

```diff
@@ -116,9 +116,13 @@ const CodeAuthorization = ({
   }, [payload])
 
   return (
-    <form onSubmit={onSubmit} ref={formRef}>
+    <form
+      onSubmit={onSubmit}
+      ref={formRef}
+      {...colorScheme.set('color', 'text')}
+    >
       {minimal ? (
-        <P>
+        <p>
```

Comment:
Yeah… this component renders a paragraph that explains what to do with the validation code. Originally it used `Interaction.P` for this, but this really doesn't work with `Flyer.P` we use elsewhere on this page. Since we use the default `p` directly below the form, I thought this might work to make the trial form usable both in article and flyer context.

Here is how it looks now:
<img width="1430" alt="Screen Shot 2022-12-13 at 09 24 26" src="https://user-images.githubusercontent.com/3907984/207263863-453c9d6a-5ab9-4bf0-bdfb-f768396e4395.png">

Compare with using `Interaction.P`:
<img width="1430" alt="Screen Shot 2022-12-13 at 09 25 18" src="https://user-images.githubusercontent.com/3907984/207264033-9725e949-2273-45c1-b9fd-e4bb982e6775.png">